src/skins/vector/css/matrix-react-sdk/views/rooms/EventTile.css

```diff
@@ -64,16 +64,14 @@ limitations under the License.
     margin-right: 95px;
 }
 
-/* all the overflow-y: hidden; are to trap Zalgos -
-   but they introduce an implicit overflow-x: auto.
-   so make that explicitly hidden too to avoid random
-   horizontal scrollbars occasionally appearing, like in
+/* Removed hidden overflow in order to have hanging elements
+   in the message stream gutters. This may re-introduce
    https://github.com/vector-im/vector-web/issues/1154
-    */
+*/
 .mx_EventTile_content {
     display: block;
-    overflow-y: hidden;
-    overflow-x: hidden;
+    /*overflow-y: hidden;
```

Comment:
This was certainly an issue in the past, however, is it possible that `overflow-x: visible` is the right fix?  (Bug #1154 was ugly scrollbar caused by overflow-x defaulting to scroll).  We have to keep the `overflow-y: hidden` (unless you want to hang elements above/below the container for some reason?) in order to trap Zalgos, which are an amusingly common phenomenon in Matrix thanks to the early geeky audience.  For instance, right now in #matrix-dev, this is the only thing stopping the top of SkaveRat's nick splurging all over Ralith:

<img width="1047" alt="screen shot 2016-07-15 at 11 06 52" src="https://cloud.githubusercontent.com/assets/1294269/16870982/4a985244-4a7c-11e6-98d8-07c2271a1d68.png">